mne/viz/evoked.py

```diff
@@ -1440,21 +1442,170 @@ def _truncate_yaxis(axes, ymin, ymax, orig_ymin, orig_ymax, fraction,
+                    colors[cond] = the_colors[color]
+                    continue
+
+    if not isinstance(colors, dict):  # default colors from M Waskom's Seaborn
+        # XXX should put a good list of default colors into defaults.py
+        colors_ = ['#e41a1c', '#377eb8', '#4daf4a', '#984ea3', '#ff7f00',
+                   '#1b9e77', '#d95f02', '#7570b3', '#e7298a', '#66a61e']
+        if len(conditions) > len(colors_):
+            msg = ("Trying to plot more than {0} conditions. We provide"
+                   "only {0} default colors. Please supply colors manually.")
+            raise ValueError(msg.format(len(colors_)))
+        colors = dict((condition, color) for condition, color
+                      in zip(conditions, colors_))
+    else:
+        colors = _aux_setup_styles(conditions, colors, "color", "grey")
+
+    # fourth, linestyles
+    if not isinstance(linestyles, dict):
+        linestyles = dict((condition, linestyle) for condition, linestyle in
+                          zip(conditions, ['-'] * len(conditions)))
+    else:
+        linestyles = _aux_setup_styles(conditions, linestyles,
+                                       "linestyle", "-")
+
+    # fifth, put it all together
+    if styles is None:
+        styles = dict()
+
+    for condition, color, linestyle in zip(conditions, colors, linestyles):
+        styles[condition] = styles.get(condition, dict())
+        styles[condition]['c'] = styles[condition].get('c', colors[condition])
+        styles[condition]['linestyle'] = styles[condition].get(
+            'linestyle', linestyles[condition])
+
+    return styles, the_colors, color_conds, color_order
+
+
+def plot_compare_evokeds(evokeds, picks=None, gfp=False, colors=None,
+                         linestyles=['-'], styles=None, cmap=None,
+                         vlines="auto", ci=0.95, truncate_yaxis=False,
+                         truncate_xaxis=True, ylim=dict(), invert_y=False,
+                         show_sensors=None, show_legend=True,
+                         split_legend=False, axes=None, title=None, show=True):
     """Plot evoked time courses for one or multiple channels and conditions.
 
-    This function is useful for comparing ER[P/F]s at a specific location. It
-    plots Evoked data or, if supplied with a list/dict of lists of evoked
-    instances, grand averages plus confidence intervals.
+    When multiple channels are passed, this function combines them all, to
+    get one time course for each condition. If gfp is True it combines
+    channels using global field power (GFP) computation, else it is taking
+    a plain mean.
+
+    This function is useful for comparing ER[P/F]s at a specific location.
+
+    It can plot a simple Evoked object or if supplied with a list or a dict
+    of lists of evoked instances, it will compute the average of the mean or
+    GFP across datasets. It can also compute some confidence intervals
+    using for example bootstrap estimates.
```

Comment:
It plots 1. a simple evoked object, 2. a list or dict of evoked objects (e.g., for multiple conditions), 3. a list or dict of lists of evokeds (e.g., for multiple subjects in multiple conditions). In the last case, it can show a confidence interval (across e.g. subjects) using parametric or bootstrap estimation.